.rubocop_todo.yml

```diff
@@ -346,6 +345,7 @@ Style/FrozenStringLiteralComment:
     - 'lib/muffin_man/fulfillment_inbound/v0.rb'
     - 'lib/muffin_man/fulfillment_inbound/v1.rb'
     - 'lib/muffin_man/listings/v20210801.rb'
+    - 'lib/muffin_man/listings/v20200901.rb'
```

Comment:
Instead of ignoring these violations, can we explore addressing them in a way that aligns with our codebase standards? This could lead to a more maintainable and cleaner code.

Entries from this file should be removed as and when we are able to fix those violations.  No new entries should be added to this file. If the rubocop rule needs to be updated, we can discuss that instead of adding new violation exceptions here.

Reply:
@ssontakke If we do not do this then we need to add new line at the to of the file, let's take this as separate action item. I will create a ticket to handle this for all versioning files listed above.